.github/workflows/nightly.yaml

```diff
@@ -91,8 +91,22 @@ jobs:
           DATE="$(date +%Y-%m-%d)"
           echo "tag=nightly-${DATE}" >> $GITHUB_OUTPUT
 
+      - name: Build Unittest Container
+        uses: docker/build-push-action@v4
+        with:
+          load: true
+          build-args: |
+            CMAKE_BUILD_TYPE=Release
+            S3GW_VERSION=${{ steps.date.outputs.tag }}
```

Comment:
Does this set radosgw --version? If so, can haz for release containers?

Reply:
No, but this will be persisted in the container image as a label named `Version`. We're doing it for release containers as well.
The container labels will survive image re-taggings, so even if you do a
`docker tag quay.io/s3gw/s3gw:v0.19.0 irq0/foo:bar`, the image will contain a `Name` label with value `s3gw` and a `Version` label with value `v0.19.0`. They can be queried e.g. from `docker inspect`